### ISSUE TITLE
Optimizer for unet model on Pascal voc

### ISSUE THREAD

**Issue by saiphanish7** [Kangyeol-Kim/Segmentation-Pytorch#5]
Can you please give me the details of optimizer you used on unet model for Pascal voc segmentation? Please also specify the loss you used and any lr_scheduler if you used it.
